src/renderer/renderer.go

```diff
@@ -1,5 +1,59 @@
 package renderer
 
+import (
+	"game"
+	"strconv"
+)
+
 func Hello() string {
 	return "Welcome to the 15 puzzle game.\n"
 }
+
+func BuildLine(startSymb string, stopSymb string, sepSymb string, tileSize int) string {
```

Comment:
IMO, the signature is far too complex. Don't allow to customize `startSymb` or `stopSymb` here. Just hard-write it. We don't allow to customize them at this point. We'll update it if our PO requires it.

Reply:
It was to avoid the duplicate code in the `BuildGrid` method.
Each line is different :
``` go
horizontalLine := BuildLine("├", "┤", "┼", size)
firstHorizontalLine := BuildLine("┌", "┐", "┬", size)
lastHorizontalLine := BuildLine("└", "┘", "┴", size)
```